### DISCUSSION TITLE
Showing Embeds fields in the list view with Admin Columns

### DISCUSSION THREAD

**Discussion by oldrup** [epiphyt/embed-privacy#210] [Show and tell]
As Embed Privacy embeds are a custom post type, Admin Columns (free) can add some interesting columns to the manage embed list. Fun? A little. Useful? I think so. Makes for a good overview of embeds available.

![image](https://github.com/epiphyt/embed-privacy/assets/17141205/eacab7f6-722d-48a5-ad17-5d53849fe730)

**Comment by MatzeKitt** [comment]
At first, I wanted to answer with a clear no because of #194, but I’m no more that sure about that (especially for easier translatability via Polylang & Co.).

Thus, I like the idea in general. We should consider that the Content can be rather large, so maybe restrict its content. But besides that, I like the idea!

**Comment by oldrup** [comment]
Admin Columns can limit the number of words shown in the content column, so the view doesn't get out of hand. 

![image](https://github.com/epiphyt/embed-privacy/assets/17141205/6fb93383-a709-4356-b667-7a586975f5e7)

And yes, if #194 becomes a reality, it's fine, but until then, I like being able to see all the privacy policy links etc at a glance, especially when adding more languages.
